src/schemas/schema.graphql

```diff
@@ -15,8 +15,8 @@ type ShopLogoUrls {
 
 "Storefront route URLs"
 type StorefrontUrls {
-  "Storefront Account Profile URL (can include `:accountId` in string)"
-  storefrontAccountProfileUrl: String
+  "Storefront Shop Profile URL (can include `:ShopId` in string)"
+  storefrontShopProfileUrl: String
```

Comment:
It looks like the word `Account` was inadvertently changed to `Shop` here?

Reply:
That's what you get when `⌘ + F`'ing blindly! Thanks for catching this.